net/ripe-atlas/Makefile

```
@@ -0,0 +1,220 @@
+	$(call LinkApplet,$(1),evntp)
+	$(call LinkApplet,$(1),evping)
+	$(call LinkApplet,$(1),evsslgetcert)
+	$(call LinkApplet,$(1),evtdig)
+	$(call LinkApplet,$(1),evtraceroute)
+	$(call LinkApplet,$(1),httppost)
+	$(call LinkApplet,$(1),onlyuptime)
+	$(call LinkApplet,$(1),perd)
+	$(call LinkApplet,$(1),rchoose)
+	$(call LinkApplet,$(1),rptaddrs)
+	$(call LinkApplet,$(1),rptra6)
+	$(call LinkApplet,$(1),rptuptime)
+	$(call LinkApplet,$(1),rxtxrpt)
+	$(call LinkApplet,$(1),telnetd)
+
+	$(INSTALL_DIR) $(1)/usr/lib/ripe-atlas/scripts
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/array.lib.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/atlas_log.lib.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/class.lib.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/common-pre.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/common.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/config.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/generic-ATLAS.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/generic-common.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/generic-reginit.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/json.lib.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/linux-functions.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/paths.lib.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/reg_servers.sh.dev
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/reg_servers.sh.test
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/reginit.sh
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/resolvconf
+	$(INSTALL_BIN) -t $(1)/usr/lib/ripe-atlas/scripts \
+					  $(PKG_INSTALL_DIR)/usr/lib/ripe-atlas/scripts/support.lib.sh
+
+	$(INSTALL_DIR) $(1)/usr/share/ripe-atlas
+	$(INSTALL_DATA) ./files/capabilities.json $(1)/usr/share/ripe-atlas/capabilities.json
+	$(INSTALL_DATA) -t $(1)/usr/share/ripe-atlas $(PKG_INSTALL_DIR)/usr/share/ripe-atlas/FIRMWARE_APPS_VERSION
+	$(INSTALL_DATA) -t $(1)/usr/share/ripe-atlas $(PKG_INSTALL_DIR)/usr/share/ripe-atlas/measurement.conf
```

This is conf, right?

```suggestion
	$(INSTALL_CONF) -t $(1)/usr/share/ripe-atlas $(PKG_INSTALL_DIR)/usr/share/ripe-atlas/measurement.conf
```